ts/parts/SVGLabel.ts

```diff
@@ -0,0 +1,573 @@
+/* *
+ *
+ *  (c) 2010-2020 Torstein Honsi
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ * */
+
+'use strict';
+
+
+import H from './Globals.js';
+import SVGElement from './SVGElement.js';
+import U from './Utilities.js';
+
+const {
+    defined,
+    extend,
+    isNumber,
+    merge,
+    removeEvent
+} = U;
+
+declare global {
+    namespace Highcharts {
+        class SVGLabel extends SVGElement {
+            public constructor(
+                renderer: SVGRenderer,
+                str: string,
+                x: number,
+                y?: number,
+                shape?: SymbolKeyValue,
+                anchorX?: number,
+                anchorY?: number,
+                useHTML?: boolean,
+                baseline?: boolean,
+                className?: string
+            );
+            public alignFactor: number;
+            public anchorX?: number;
+            public anchorY?: number;
+            public baseline?: boolean;
+            public baselineOffset: number;
+            public bBox: BBoxObject;
+            public box?: SVGElement;
+            public className?: string;
+            public deferredAttr: SVGAttributes;
+            public fill: ColorType;
+            public height: number;
+            public heightSetting: number;
+            public needsBox?: boolean;
+            public padding: number;
+            public paddingLeft: number;
+            public shape?: SymbolKeyValue;
+            public stroke: number|string;
+            public 'stroke-width': string;
+            public symbolKey?: SymbolKeyValue | string;
+            public text: SVGElement;
+            public textAlign: string;
+            public textStr?: string;
+            public width: number;
+            public widthSetting?: number;
+            public x?: number;
+            public xSetting: number;
+            public y?: number;
+            public ySetting: number;
+        }
```

Comment:
This should be replaced with `let SVGLabel: SVGLabel;` as it is only for backwards compatibility. Instead use `import type SVGLabel from './SVGLabel';` in other files to reference the type and save real imports.

Reply:
Could I ask you to have a look at this?